pkg/entities/message_repost_history.go

```diff
@@ -16,11 +17,23 @@ func (e *Entity) IsRepostableMessage(req request.CreateMessageRepostHistRequest)
 	return false
 }
 
-func (e *Entity) CreateRepostMessageHist(req request.CreateMessageRepostHistRequest, repostChannelID string) error {
-	return e.repo.MessageRepostHistory.CreateIfNotExist(model.MessageRepostHistory{
-		GuildID:         req.GuildID,
-		OriginMessageID: req.MessageID,
-		OriginChannelID: req.ChannelID,
-		RepostChannelID: repostChannelID,
-	})
+func (e *Entity) CreateRepostMessageHist(req request.CreateMessageRepostHistRequest, repostChannelID string) (*model.MessageRepostHistory, error) {
```

Comment:
can we add `RepostChannelID` as a property in `request.CreateMessageRepostHistRequest`
=>
```
func (e *Entity) CreateRepostMessageHist(req request.CreateMessageRepostHistRequest) (*model.MessageRepostHistory, error) {
```